dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/shim/POJO.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal_fat.apps.shim;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+import io.opentelemetry.instrumentation.annotations.WithSpan;
+
+import io.opentracing.Span;
+import io.opentracing.Tracer;
+
+import org.junit.Assert;
+
+/**
+ * POJO to test Traced annotation.
+ */
+@ApplicationScoped
+public class POJO {
+    /**
+     * Method that we expect to be Traced implicitly.
+     */
+    @WithSpan
+    public void annotatedClassMethodImplicitlyTraced(Tracer tracer) {
+        System.out.println("Called annotatedClassMethodImplicitlyTraced");
+        Span span = tracer.activeSpan();
+        Assert.assertNotNull(span);
+    }
```

Comment:
It would be better if we could test a little more here.

In this test can we check that the spanId returned by the opentracing shim is the same as the one returned by OpenTelemetry?

It would be better if we had a test which called the opentracing API a bit more, perhaps creating a span and seeing that it is exported correctly using `InMemorySpanExporter`.